addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/MainTVServerService.java

```diff
@@ -130,7 +132,7 @@ public void run() {
 
                     updateResourceState("MainTVAgent2", "GetCurrentBrowserURL", null);
                 } catch (Exception e) {
-                    logger.debug("Exception during poll : {}", e);
+                    reportError("Error occurred during poll", e);
```

Comment:
I forgotten to say, that binding including your changes works fine at least with my TV.